pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -735,7 +735,15 @@ impl LayerInner {
 
         LayerInner {
             conf,
-            path_str: path.to_string().into(),
+            debug_str: {
+                format!(
+                    "tenants/{}/timelines/{}/{}",
```

Comment:
nit: if this is always logged from the context of a particular tenant, we could also leave off the `tenants/{}` part (assuming the tenant is always in the tracing span where we log it)

Reply:
I'll need to double check the error message

Reply:
we'll need to keep the timeline id because it might come from an ancestor. fixed in https://github.com/neondatabase/neon/pull/7565/commits/e1a1ab74d75948372faadae53e4e52cdfbce1339